index.php

```diff
@@ -132,15 +140,47 @@
             $url = sprintf('team/%s', $platform);
             break;
     }
+    return $verifyUrl($url) ? $routes[$url] : $routes['default'];
+};
+
+$langUrlBuilder = function ($url, $platform) use ($verifyUrl, $routes) {
+    if ($verifyUrl($url)) {
+        return $routes[$url] . "/" . $platform;
+    } else {
+        return $routes['default'];
+    }
+};
+
+$langs = ['ar', 'bg', 'bs', 'cs', 'de', 'el', 'en', 'es', 'fr', 'hr', 'hu', 'id', 'ja', 'pl', 'pt', 'ro', 'ru', 'tr', 'uk', 'vi', 'zh'];
 
```

Comment:
https://support.crowdin.com/api/status/

We can do it with this API call. I will set something up so that the `$langs` gets updated from crowdin making it pretty much transparent when we deploy. A `deploy` script could do the API call and `git pull` and we will be good to go